app/service/FileStoreService.scala

```diff
@@ -106,7 +108,7 @@ class FileStoreService @Inject() (connector: FileStoreConnector) extends Logging
     connector.delete(fileId)
 
   private def toFileAttachment(size: Long): FileMetadata => FileStoreAttachment = { r =>
-    FileStoreAttachment(r.id, r.fileName, r.mimeType, size)
+    FileStoreAttachment(r.id, r.fileName.getOrElse(""), r.mimeType.getOrElse(""), size)
   }
```

Comment:
You might be able to get rid of this method entirely as I don't think the `upload` method above is used anymore